package.json

```diff
@@ -45,7 +45,8 @@
   "dependencies": {
     "@types/form-data": "^2.5.0",
     "async-iterator-to-array": "0.0.1",
-    "dcl-catalyst-commons": "^5.0.0",
+    "cookie": "^0.4.1",
+    "dcl-catalyst-commons": "tag-add-cookies-v7",
```

Comment:
this will be a release version, right?

Reply:
Yes! This is blocked by https://github.com/decentraland/catalyst-commons/pull/68